Readme.md

```diff
@@ -1,6 +1,41 @@
-# Data
+# BAHIS Data Processing Pipelines
+
+## Data
 Available here: https://drive.google.com/drive/folders/1l5AL2O7nnpnsh73UjqjLYcBotwdthsWs?usp=sharing
 
-# Comment
+## Comment
 I've pulled data from static bahis and had some fun formatting it (all the different forms submissions are in one table with the actual data stored as key:value pairs). I have uploaded the data for you into google drive, those are files in a format `formdata_*.csv`.
 You will notice however that it is not possible to find out what different indexes mean in many cases. I leave that for you to find out as you go through whichever dataset you find most interesting. The tables that contain keys for submitted data are in `input/STATICBAHIS*.csv` files, though at this point I'm not sure what is what.
+
+##  Setting up the environment for data pipeline for Ubuntu
+
+###  Install python 3.10 or later versions
```

Comment:
This is not how I would recommend doing this. First, it's very Ubuntu-specific and I think you might be the only Ubuntu user :wink: and, second, it's not the best way to do it Ubuntu (IMHO!).

I would suggest that we make use of this pipenv feature: https://pipenv.pypa.io/en/latest/advanced/#automatic-python-installation

Basically, you need to install pyenv from your normal repositories (no PPAs required and true for Arch Linux, macOSX and other flavours of unix... and _possibly_ translates over to Windows (https://pyenv-win.github.io/pyenv-win/)). And then pipenv should do the rest.

Reply:
I'm not sure we need to get into details in this part as it seem to get messed up easily. E.g. currently on ubuntu (arm)  pipenv installed from distribution packages doesn't really work so it has to be installed from `pip`. 

I think for installation of such high level things like python and pipenv we should refer to official documentation and not worry about system-specific hacks.